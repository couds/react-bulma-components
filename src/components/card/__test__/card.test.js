import React from 'react';
import renderer from 'react-test-renderer';
import Card from '..';

describe('Card component', () => {
  it('Should have card classname', () => {
    const component = renderer.create(
      <Card>Card Content</Card>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
  it('Should have card-image classname', () => {
    const component = renderer.create(
      <Card type="image" size="4by3" src="http://bulma.io/images/placeholders/1280x960.png" />,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
  it('Should have card-content classname', () => {
    const component = renderer.create(
      <Card type="content">Content</Card>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
