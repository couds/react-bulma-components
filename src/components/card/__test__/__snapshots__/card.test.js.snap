// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component Should have card classname 1`] = `
<div
  className="card"
  style={Object {}}
>
  Card Content
</div>
`;

exports[`Card component Should have card-content classname 1`] = `
<div
  className="card-content"
  style={Object {}}
>
  Content
</div>
`;

exports[`Card component Should have card-image classname 1`] = `
<div
  className="card-image"
  style={Object {}}
>
  <figure
    className="image is-4by3"
    style={Object {}}
  >
    <img
      alt=""
      onError={[Function]}
      src="http://bulma.io/images/placeholders/1280x960.png"
    />
  </figure>
</div>
`;
